packages/dotcom-shared/src/types.ts

```diff
@@ -169,8 +169,8 @@ export type ZErrorCode = keyof typeof ZErrorCode
 
 // increment this to force clients to reload
 // e.g. if we make backwards-incompatible changes to the schema
-export const Z_PROTOCOL_VERSION = 2
-export const MIN_Z_PROTOCOL_VERSION = 2
+export const Z_PROTOCOL_VERSION = 3
+export const MIN_Z_PROTOCOL_VERSION = 3
```

Comment:
We need to do this in two steps otherwise it will force literally everybody to reload when this gets deployed.

1. bump the Z_PROTOCOL_VERSION and always use the new user init
2. several days later, bump the MIN_Z_PROTOCOL_VERSION so the backend starts rejecting old clients.